openpgp/packet/signature.go

```diff
@@ -774,14 +841,19 @@ type outputSubpacket struct {
 func (sig *Signature) buildSubpackets() (subpackets []outputSubpacket, err error) {
 	creationTime := make([]byte, 4)
 	binary.BigEndian.PutUint32(creationTime, uint32(sig.CreationTime.Unix()))
-	subpackets = append(subpackets, outputSubpacket{true, creationTimeSubpacket, false, creationTime})
+	subpackets = []outputSubpacket{
+		outputSubpacket{true, creationTimeSubpacket, false, creationTime},
+	}
```

Comment:
(Why this change?)

Reply:
(To improve readability (and explicit initialization, not appending to nil))

Reply:
But the value is already initialized as an out parameter, to the nil slice, which in golang kinda acts as an empty slice, and it's legal to append to it. Initializing it twice seems a bit redundant to me.

Reply:
Yes, I know it is legal to append to the nil slice. Indeed, I missed that it is already initialized in the function signature